modules/ListView.js

```diff
@@ -430,7 +464,8 @@
      */
     __generateChildArgs: function(model) {
       var args = {
-        'context': _.extend({}, _.result(this, '__childContext'))
+        'context': _.extend({}, _.result(this, '__childContext')),
+        'listView': this
       };
```

Comment:
no where by torso. But if you want a child view to listen to events like "reorder" and other listview communication, i thought it couldn't hurt and could be very useful